lib/android_build/maesdk/src/main/java/com/microsoft/applications/events/HttpClient.java

```diff
@@ -306,6 +310,14 @@ private String calculateID(android.content.Context context) {
     }
   }
 
+  private String getDeviceClass(android.content.Context context) {
+    if (context.getResources().getConfiguration().isLayoutSizeAtLeast(Configuration.SCREENLAYOUT_SIZE_LARGE)) {
+      return "Android.PC";
+    } else {
+      return "Android.Phone";
```

Comment:
> What's the source for this and how are these values determined, is the list of values complete and why are you not expecting tablet here?

@saurabh-sagrawal  - the list is coming from common schema Part A extension:

https://github.com/microsoft/common-schema/blob/main/PartA/device.md#deviceclass


<html>
<body>
<!--StartFragment-->

iOS | iOS.Emulator iOS.Phone iOS.Tablet iOS.Phablet iOS.AppleTV
-- | --
Android | Android.Phone Android.PC

<!--EndFragment-->
</body>
</html>

Reply:
Thanks lalitb for help to reply! 
And I have made the related change for 1.

Reply:
It's cool, my minor suggestion would be to revisit / review this in schema forum. These definitions were just someone's random suggestion, and may not be very neatly tuned / optimal for today's realities as of 2021... i.e. `Android.PC` - what is that exactly, like a Chromebook or a Tablet with external keyboard attached??

Reply:
It may a random suggestion, but it may hard to change these values right now, the  low version Edge app uploads these strings, and the other apps may have the similiar situation, we may need to keep them consistent.